packages/lexical-devtools-core/src/generateContent.ts

```diff
@@ -396,6 +398,21 @@ function printTitleProperties(node: LinkNode) {
   return str;
 }
 
+function printStateProperties(node: LexicalNode) {
+  if (!node.__state) {
+    return false;
+  }
+  const states = [];
+  for (const [stateType, value] of node.__state.knownState.entries()) {
+    states.push(`[${stateType.key}: ${value}]`);
+  }
+  let str = states.join(',');
+  if (str != null) {
```

Comment:
The null check isn't going to do anything

```suggestion
  if (str !== '') {
```